### ISSUE TITLE
Add labels for form elements that don't have them

### ISSUE THREAD

**Issue by teshsopher** [tl-its-umich-edu/remote-office-hours-queue#159]
3 form elements currently don't have a label. In each case, the text input has no label, but, instead, has placeholder which isn't accessible.

Labels (even if it's just screen reader only text) need to be added to:
- Add Host
- Add Attendee
- Add Queue

**Comment by teshsopher** [comment]
Tested in QA: 
- As an attendee in queue, when the Agenda text field is opened, the text field doesn't have a label
- The Name and Description fields in the Queue Settings page don't have labels 
- The Add Host field in the Queue Settings page doesn't have a label

For these, I think, visually, these are labelled, it's just that screen readers don't "see" the label, so the fields don't appear to have one. Adding an aria-label attribute should work here

**Comment by ssciolla** [comment]
`aria-label` needs to be moved from the containing `form` element/component to the specific `input` element.

**Comment by teshsopher** [comment]
Tested and working in QA
